float64 sail
float64 rudder
float64 u
float64 v
float64 r
float64 dheel
float64 x
float64 y
float64 heading
float64 heel
float64 aws
float64 awa
